cli/pcluster/cfnconfig.py

```diff
@@ -135,6 +108,12 @@ def __init__(self, args):
         except AttributeError:
             pass
 
+    @staticmethod
+    def __fail(message):
+        """Print an error message and exit."""
+        print("ERROR: {0}".format(message))
```

Comment:
Use `LOGGER.critical()` as it logs to both stdout and the cli log file `~/.parallelcluster/pcluster-cli.log`

Reply:
Also I've defined a `__fail()` function in https://github.com/aws/aws-parallelcluster/pull/858

Reply:
I know you already defined the `__fail()` function. Let's rebase it when required.